index.html

```diff
@@ -151,11 +151,21 @@ <h2 class="bannerbox__title">I need a ride</h2>
 
                             <div class="form-column">
                                 <div class="form-group">
+                                    <label for="riderPickUpAddress">Pick up address</label>
+                                    <input type="text" class="form-input form-input--medium" id="riderCollectionAddress" placeholder="Your pick up address" name="RiderCollectionAddress" required>
```

Comment:
Can you please remove the `form-input--medium` class from both of these new address fields? It's more important that they are appropriately sized than that they align with the fields around them.